.travis.yml

```diff
@@ -5,18 +5,20 @@ dist: trusty
 sudo: required
 addons:
     chrome: stable
+services:
+   - docker
 cache:
     npm: true
     directories:
         - node_modules
 before_script:
+  - docker swarm init
+  - docker stack deploy -c docker-compose-api.yml docks-api
```

Comment:
Will the Stack be fully deployed by the time we start running the tests? From experience Docker CLI will return before the services are operational

Reply:
On my fork this worked; I recall you used a script to wait though - if necessary I'll add that.

Reply:
It might be worth running the `wait-for-docks.sh` script before starting the testing. The script should return when the API and the DB are done initializing.

https://github.com/TripleParity/docks-api/blob/develop/integration-test.sh
https://github.com/TripleParity/docks-api/blob/develop/wait-for-docks.sh